components/monitoring/logging/base/configure-logging/configure-logforwarder.yaml

```diff
@@ -14,10 +14,26 @@ spec:
       secret:
         name: cloudwatch-forwarder
       type: cloudwatch
```

Comment:
Adding the following in `base/` will make it apply to production clusters as well, I thing this whole thing needs to be in the staging directory for now, otherwise we'll need a patch file to remove this from production